AABB_tree/include/CGAL/AABB_tree.h

```diff
@@ -156,7 +156,31 @@ namespace CGAL {
     /// `build()` explicitly to ensure that the next call to
     /// query functions will not trigger the reconstruction of the
     /// data structure.
+    /// The parameter pack `T` is of any types
+    /// such that `Primitive` has a constructor with the following signature:
+    /// `Primitive(%InputIterator, T...)`. A call to `AABBTraits::set_shared_data(t...)`
```

Comment:
no need to mention primitive here

Reply:
How do I explain `T` then ?

Reply:
With [`set_shared_data()`](https://doc.cgal.org/latest/AABB_tree/classAABBTraits.html#a6dba5130cc78298fb7039f479f657fe4) directly since it is was it is called in practice.